### PR TITLE
Add stylelint

### PR DESCRIPTION
**Pull request by uladzimirdev**
### Description

Introduce stylelint for css files.

if you think you already saw this PR, you're right https://github.com/metabase/metabase/pull/33033

### How to verify

- add duplicated line to the css file
- try to commit it
- lint-staged should fail

### CODE REVIEW

File: frontend/src/metabase/components/Popover/Popover.module.css [hunk 1]
```diff
@@ -51,7 +52,6 @@
 
 :global(.tippy-box[data-theme~="popover"]) {
   font-size: inherit;
-  background-color: var(--color-bg-white);
```
Comment (uladzimirdev) [inline frontend/src/metabase/components/Popover/Popover.module.css:54]: duplicated

File: frontend/src/metabase/css/admin.module.css [hunk 2]
```diff
@@ -66,8 +66,6 @@
 
 :global(.AdminList-search .Icon) {
   position: absolute;
-  margin-top: auto;
```
Comment (uladzimirdev) [inline frontend/src/metabase/css/admin.module.css:69]: `margin: auto` overrides it

File: frontend/src/metabase/css/components/form.module.css [hunk 3]
```diff
@@ -36,9 +39,6 @@
   color: var(--color-text-dark);
   background-color: var(--color-bg-white);
   padding: 0.75em;
-}
-
-:global(.Form-input) {
```
Comment (uladzimirdev) [inline frontend/src/metabase/css/components/form.module.css:41]: merged 2 declarations

File: frontend/src/metabase/css/query_builder.module.css [hunk 4]
```diff
@@ -270,12 +276,6 @@
   padding-right: 10px;
 }
 
-/* DATA SECTION */
-
-:global(.GuiBuilder-data) {
```
Comment (uladzimirdev) [inline frontend/src/metabase/css/query_builder.module.css:275]: duplicated

File: frontend/src/metabase/visualizations/components/LineAreaBarChart.module.css [hunk 5]
```diff
@@ -188,11 +193,7 @@
   transition: opacity 0.25s linear;
 }
 
-:global(.LineAreaBarChart .dc-chart .event-line) {
```
Comment (uladzimirdev) [inline frontend/src/metabase/visualizations/components/LineAreaBarChart.module.css:191]: duplicated

File: frontend/src/metabase/visualizations/components/LineAreaBarChart.module.css [hunk 6]
```diff
@@ -234,9 +235,6 @@
 
 :global(text.value-label) {
   pointer-events: none;
-}
-
-:global(text.value-label) {
```
Comment (uladzimirdev) [inline frontend/src/metabase/visualizations/components/LineAreaBarChart.module.css:239]: duplicated

**Review by ranquild — APPROVED** [review]

**Pull request merged**